internal/controller/clusterinstance_helper.go

```diff
@@ -39,17 +39,17 @@ type SpecialVars struct {
 	ControlPlaneAgents, WorkerAgents int
 }
 
-// SiteData is a special object that provides an interface to the SiteConfig spec fields for use in rendering templates
-type SiteData struct {
-	Site        v1alpha1.SiteConfigSpec
+// ClusterData is a special object that provides an interface to the SiteConfig spec fields for use in rendering templates
```

Comment:
Missed SiteConfig in the comment if you care.

Reply:
This is updated in the second commit.